FakebookNotifications/FakebookNotifications.WebApi/Hubs/NotificationHub.cs

```diff
@@ -179,19 +197,34 @@ public async Task SendAll(string user, string notification)
             await Clients.All.SendAsync("SendAll", user, notification);
         }
 
-        //Send a notification to all clients within a specific group
+        /// <summary>
+        /// Send a notification to all clients within a specific group. Can be used to send a notification to
+        /// all connected devices of the same user.
+        /// </summary>
+        /// <param name="group">group name to send notifications to</param>
+        /// <param name="notification">notification to be sent as string</param>
+        /// <returns></returns>
```

Comment:
there's boilerplate we can put here for any Task returns, run a search for "c# xml documentation Task async returns" or something like that